test/client/spec/victory-util/add-events.spec.js

```diff
@@ -0,0 +1,95 @@
+/* eslint max-params: 0 */
+
+import React from "react";
+import { forEach, get } from "lodash";
+import { mount } from "enzyme";
+import { addEvents } from "src/index";
+import { MockChart, MockLabel, MockDataComponent } from "../mock-components";
+
+describe("victory-util/add-events", () => {
+  const EventedMockChart = addEvents(MockChart);
+
+  const expectEventsTriggered = (scopeFn, testFn, expectations, wrapper) => {
```

Comment:
I'm having a hard time following what is being asserted with this. We're using sinon / sinon-chai to spy on / write expectations about functions elsewhere. Could you use that here instead?